tests/conftest.py

```diff
@@ -308,7 +308,12 @@ def test_workspace(
         if e.response.status_code == 409:
             logger.info("Test workspace already exists")
             workspace = test_config_manager.get_workspace()
-            yield WorkspaceMetadataResponse(**workspace)
+            if not workspace:
+                raise ValueError(
+                    "Unexpected error when trying to get workspace. It either doesn't exist, or there was a conflict."
+                    "Please check the logs for more information, or try viewing the database directly."
```

Comment:
```suggestion
                    "Please contact us for support (Cloud) or check the logs for more information (self-hosted)."
```

Reply:
This is conftest - i wrote this message more for devs